bqschema_suite_test.go

```diff
@@ -1,10 +1,10 @@
-package bqschema_test
+package bqschema
```

Comment:
When trying to apply the change, I came up with another reason why it works better to have the tests *not* use a dot import: I cannot produce a merge-able commit that you can just click the button on. I'm forced to commit with `import . "github.com/nbio/bqschema"` in my repo, and you would have to merge by hand, fixing the import path.

This complication does not arise when the tests are in `package bqschema`.